tests/Microsoft.Bot.Builder.Azure.Tests/BlobsStorageTests.cs

```diff
@@ -34,7 +35,12 @@ public BlobsStorageTests(ITestOutputHelper testOutputHelper)
 
         protected override string ContainerName => $"blobs{_testName.ToLower().Replace("_", string.Empty)}";
 
-        public async void Dispose()
+        public async Task InitializeAsync()
```

Comment:
Is there a better way to do this? `InitializeAsync()` is required by `IAsyncLifetime`, but not actually needed for this test.